.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing to cloudone-filestorage-plugins
+
+:tada: Thank you for taking the time to contribute! :tada:
+
+## How Can I Contribute
+
+- Fork this repository
+- Commit your code with message following [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/).
+- Submit a PR and request review from @trendmicro/cloud-one-file-storage-security
```

Comment:
Submit a PR and request review from @trendmicro/cloud-one-file-storage-security -> Submit a PR and request a review from @trendmicro/cloud-one-file-storage-security.

(note 'a' and '.')